src/price.css

```diff
@@ -0,0 +1,18 @@
+.price {
+  display: flex;
+  align-items: center;
+  margin-top: 23px;
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+  margin-right: 86px;
+  color: var(--secondary-color);
+}
+
+.price_number {
```

Comment:
Consider renaming the class 'price_number' to 'product-card__price-number' to maintain consistency with BEM naming conventions.